cellxgene_schema_cli/tests/fixtures/examples_ontology_test.py

```diff
@@ -4,22 +4,21 @@
 invalid_species = ["Caenorhabditis elegans"]
 
 valid_genes = {
-    ontology.SupportedOrganisms.HOMO_SAPIENS: {"ENSG00000141510": ("TP53", 5676)},
+    ontology.SupportedOrganisms.HOMO_SAPIENS: {"ENSG00000141510": ("TP53", 6836)},
```

Comment:
for the changes to gene lengths i'm just trusting that the new gencode files have the correct lengths so i just updated all the test fixtures to whatever the test expected when it failed.

Reply:
I think that makes sense